### PR TITLE
Add 7.1.0-alpha.6 changelog

### PR DESCRIPTION
**Pull request by wadells**
## Description
Add 7.1.0-alpha.6 changelog

## Type of change
* Documenation

## TODOs
- [ ] Address review feedback

## Testing Done
Eyeballed the links and format here: https://github.com/gravitational/gravity/blob/c1e4524e614573ea38be46f2cd972aa735db7f7d/docs/7.x/changelog.md

### CODE REVIEW

File: docs/7.x/changelog.md [hunk 1]
```diff
@@ -54,6 +54,21 @@ extend updates past End of Support through customer agreements if required.
...
+* Improve performance of AWS autoscaling integration ([#2458](https://github.com/gravitational/gravity/pull/2458))
+* Improve satellite cluster health monitoring on large clusters up to 1000 nodes ([#2439](https://github.com/gravitational/gravity/pull/2439), [planet#828](https://github.com/gravitational/planet/pull/828), [satellite#297](https://github.com/gravitational/satellite/pull/297))
+
+#### Bugfixes
+* Fix ownership of monitoring and logrange directories ([#2448](https://github.com/gravitational/gravity/pull/2448))
+* A kubernetes config is no longer required for `gravity app sync` ([#2438](https://github.com/gravitational/gravity/pull/2438))
+
+#### Internal Changes
+* Upgrade internal kubernetes client go to v1.19.8 and helm to v3.4.2. Removed helm2. ([#2433](https://github.com/gravitational/gravity/pull/2433), [planet#827](https://github.com/gravitational/planet/pull/827), [rigging#100](https://github.com/gravitational/rigging/pull/100), [satellite#299](https://github.com/gravitational/satellite/pull/299))
```
Comment (wadells) [inline docs/7.x/changelog.md:69]: @bernardjkim I may have made up that helm 2 was removed.  Can you check this note for me, please?

File: docs/7.x/changelog.md [hunk 2]
```diff
@@ -54,6 +54,21 @@ extend updates past End of Support through customer agreements if required.
 
 7.1 is currently pre-release.
 
+### 7.1.0-alpha.6 (April 14, 2021)
+
+#### Improvements
+* Add GCE Alias IP support for flannel ([#2439](https://github.com/gravitational/gravity/pull/2439), [planet#830](https://github.com/gravitational/planet/pull/830), [flannel#10](https://github.com/gravitational/flannel/pull/10))
```
Comment (knisbet) [inline docs/7.x/changelog.md:60]: I think we put `.` at the end of each entry, although I think alpha.5 is also missing this.

**Review by knisbet — APPROVED** [review]

File: docs/7.x/changelog.md (hunk 1 above)
Comment (bernardjkim) [reply docs/7.x/changelog.md:69]: Helm v2 binary is still included in gravity. The integrated helm/app command has just been upgraded to use helm v3

Comment (bernardjkim) [reply docs/7.x/changelog.md:69]: I still don't really know the use cases, so I'm not sure if we can just remove helm v2 binary or not.

File: docs/7.x/changelog.md (hunk 2 above)
Comment (wadells) [reply docs/7.x/changelog.md]: Added to this changelog in 6aa4ac2.  I'll catch the others in a later PR as I don't want to include unrelated changes from a diff tracking perspective.

File: docs/7.x/changelog.md (hunk 1 above)
Comment (wadells) [reply docs/7.x/changelog.md]: Helm 2 is no longer mentioned in 6aa4ac2.  My impression is without tiller, the helm 2 binary is somewhat useless.  Customers would have to run their own tiller (in which case they could probably bootstrap their own helm 2 in the hooks containers).

If we do remove helm 2, it should be before 7.1.0 releases.  It'd be bad form to remove something like that in a patch release.

Comment (bernardjkim) [reply docs/7.x/changelog.md]: Hmm, that's a good point. I'll do some clean up before 7.1.0 release.

**Pull request merged**